spec/post_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do

```

Comment:
- [ ] All methods in your Post model should be tested as mentioned in the project requirements. Please write a test for the methods `latest_comments` and `update_counter_posts`